src/page-templates/content-page/content-page-template.tsx

```diff
@@ -0,0 +1,503 @@
+                setFeedbackId(data._id);
+            });
+        setRatingStars(stars); // This should update both rating sections but it doesn't.
+        setFeedbackModalStage(stars === 3 ? 'text' : 'checkbox');
+    };
+
+    const ratingSection = (
+        <div sx={styles.ratingSection}>
+            <span>Rate this {normalizeCategory(category)}</span>
+            <ContentRating stars={ratingStars} onRate={onRate} />
+        </div>
+    );
+
+    const renderExternalExamples = (addtlStyles: ThemeUICSSObject = {}) => (
+        <div sx={{ ...styles.externalExamples, ...addtlStyles }}>
+            {githubUrl && (
+                <Button
+                    hasIcon
+                    href={githubUrl}
+                    target="_blank"
+                    variant="secondary"
+                    iconName={ESystemIconNames.EXTERNAL}
+                    sx={{ width: 'auto' }}
+                >
+                    View Code
+                </Button>
+            )}
+            {liveSiteUrl && (
+                <Link
+                    linkIcon="arrow"
+                    href={liveSiteUrl}
+                    sx={{ marginTop: ['inc30', 0] }}
+                >
+                    Try it
+                </Link>
+            )}
+        </div>
+    );
+
+    const contentFooter = (
+        <div sx={styles.footer}>
+            <div>
+                <HorizontalRule />
+                {!previewMode && (
+                    <div sx={styles.footerActions}>
+                        <SocialButtons
+                            description={parseUndefinedValue(description)}
+                            heading={title}
+                            authors={authors}
+                            tags={tags}
+                        />
+                        {ratingSection}
+                    </div>
+                )}
+            </div>
+            {series && <SeriesCard series={series} currentTitle={title} />}
+            <div>
+                <TypographyScale
+                    variant="heading5"
+                    sx={{ marginBottom: 'inc30' }}
+                >
+                    Related
+                </TypographyScale>
+                <Grid gap={['inc30', null, 'inc40']} columns={[1, null, 2]}>
+                    {relatedContent.map(piece => (
+                        <Card
+                            sx={{ height: '100%' }}
+                            key={piece.slug}
+                            {...getCardProps(piece, 'related')}
+                        />
+                    ))}
+                </Grid>
+            </div>
+            {hasRequestContentFlow && (
+                <Button
+                    variant="secondary"
+                    sx={styles.requestBtn}
+                    onClick={() => setModalStage('text')}
+                >
+                    {requestButtonText}
+                </Button>
+            )}
+        </div>
+    );
+
+    function VideoOrPodcastContent() {
```

Comment:
Very nitpicky, but any reason for using these function declarations instead of arrow function? Just pointing it out since there are already a lot of arrow functions in this file.